src/main/resources/META-INF/maven/site-macros.vm

```diff
@@ -143,12 +146,12 @@
 #*    *##set( $style = "" )
 #*  *##end
 #*  *##set( $style = "$width$height$style" )
-<img class="$class"$src$alt#if( $style ) style="$style"#end />##
+<img#if( $class ) class="$class"#end$src$alt#if( $style ) style="$style"#end />##
 #end
 ##
 #macro( banner $banner $id )
 #**##if( $banner )
-#*  *#<div id="$id"><h1>#link( $banner false false false "set" )</h1></div>##
+#*  *#<div id="$id"><h1>#link( $banner false false false "set" false "" )</h1></div>##
```

Comment:
I would rather use `$banner null false null null null "set" ""` (slightly clearer). Using `false` for String arguments is confusing to me.